### PR TITLE
Create DBRS guide

### PR DESCRIPTION
**Pull request by bd-dean**
Can't merge until updates to links once release has been confirmed.

### CODE REVIEW

File: source/guides/stacks/dbrs.rst [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+   :header-rows: 1
+
+   * - Channel 2
+     - Channel 2
+     - Channel 1
+     - Channel 1
+     - Channel 0
+     - Channel 0
+
+    * - Slot 1
```
Comment (mvincerx) [inline source/guides/stacks/dbrs.rst]: Move back one space all lines from 65 to 77. It's Python. White space matters.  😉

File: source/guides/stacks/dbrs.rst [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+=========================
+
+The persistent memory DIMMs can be configured in devdax or fsdax mode. The use case to enable database stack on a kubernetes environment currently only support fsdax mode.
+
+Configuration Steps
+===================
+
+.. important::
+
+Run the following steps with root privileges (sudo) as shown in the examples
```
Comment (mvincerx) [inline source/guides/stacks/dbrs.rst]: Left-align "Run.." with start of important above it.

File: source/guides/stacks/dbrs.rst [hunk 3]
```diff
@@ -0,0 +1,599 @@
...
+   .. code-block:: bash
+
+      sudo mkfs.ext4 /dev/pmem{n}
+      sudo mount -o dax /dev/pmem0 /mnt/dax0
+
+
+Running DBRS with Cassandra
+***************************
+
+DBRS with Cassandra can be deployed as a standalone container or inside Kubernetes. Instructions for both cases is included here. Note that you can use the released `Docker image`_.  These instructions are to provide a baseline for creating your own container image.
```
Comment (mvincerx) [inline source/guides/stacks/dbrs.rst]: Add a corresponding URL at bottom for ``Docker image``_.

File: source/guides/stacks/dbrs.rst (hunk 1 above)
Comment (bd-dean) [reply source/guides/stacks/dbrs.rst]: Done. Removed 12 whitespaces.

File: source/guides/stacks/dbrs.rst (hunk 2 above)
Comment (bd-dean) [reply source/guides/stacks/dbrs.rst]: Aligned.  Don't know how I missed that.

File: source/guides/stacks/dbrs.rst (hunk 3 above)
Comment (bd-dean) [reply source/guides/stacks/dbrs.rst]: Added URL but it's blank until release is official.

Comment (bd-dean) [reply source/guides/stacks/dbrs.rst]: And in next commit fixed the whitespace I put in the link name by mistake...

**Comment by bd-dean** [comment]
@mvincerx links added, content reviewed by stakeholders, should be ready to go.

**Comment by mvincerx** [comment]
Looks really good, @bd-dean. There's some fussy trademarks that need to be added. 
Do you mind if I push to this PR?

**Comment by mvincerx** [comment]
Hold on all commits, please. I'm pushing to this PR.

**Comment by mvincerx** [comment]
LGTM. Will release per Swetha's timeline. 🤐

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**